Microsoft.CAP.Portal/Microsoft.CAP.Portal/Controllers/HomeController.cs

```diff
@@ -1,20 +1,46 @@
-﻿using Microsoft.CAP.Portal.Models;
+﻿using Microsoft.CAP.Model.Schema.API;
+using Microsoft.CAP.Portal.Models;
+using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
 using System.Web;
 using System.Web.Mvc;
 
 namespace Microsoft.CAP.Portal.Controllers
 {
```

Comment:
A best practice is always put all using indie namespace, and simplify those using prefix